db/schema.rb

```diff
@@ -79,6 +79,8 @@
     t.datetime "updated_at", precision: 6, null: false
     t.string "code"
     t.uuid "district_id", null: false
+    t.boolean "archive", default: false
```

Comment:
This looks like a typo. Rollback the current migration and drop `archive` field from lsg_body

Reply:
Done.